### PR TITLE
Updated makefile to install the latest node in the ppa, currently 6

### PR DESCRIPTION
**Pull request by hatched**
Now that the ppa has been updated to node 6 make sure that it always installs the latest.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

`make sysdeps`
Make sure that the versions have been updated to 6 and 3 respectively.
`node --version`
`npm --version`
Clean everything and make sure it builds and runs properly.
`make clean-all deps fast-babel test-js test-js-old lint-js run`
Then do a quick run through of the GUI to make sure it works as expected.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5145/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -49,9 +49,8 @@ STATIC_CSS_FILES = \
 
 LSB_RELEASE = $(shell lsb_release -cs)
 
-# The nodejs must be versioned in order to get the correct one from the PPA.
 # libfontconfig1 is required by phantom.
-SYSDEPS = nodejs=0.10.37-1chl1~utopic1 python-virtualenv g++ inotify-tools libfontconfig1
```
Comment (jrwren) [inline Makefile]: We might want to pin it to 6.2.0 here so that it errors if the PPA failed to be configured for some reason.

**Comment by hatched** [comment]
Because the ppa no longer contains the old version of node this branch needs to land before anyone tries to build the GUI from scratch :shipit:

**Comment by jrwren** [comment]
👍

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**